File: ping-pong-grpc/src/main/java/net/serg/client/PingPongClient.java
package net.serg.client;

import net.serg.server.PingPongServer;
import net.serg.stubs.pingpong.PingPongServiceGrpc;
import net.serg.stubs.pingpong.PingRequest;
import io.grpc.ManagedChannel;
import io.grpc.ManagedChannelBuilder;
import net.serg.stubs.pingpong.PongResponse;

import java.util.logging.Logger;

public class PingPongClient {

    private static final Logger logger = Logger.getLogger(PingPongClient.class.getName());
    public static void main(String[] args) {
        ManagedChannel channel = ManagedChannelBuilder.forAddress("localhost", 8080)
                                                      .usePlaintext()
                                                      .build();

        PingPongServiceGrpc.PingPongServiceBlockingStub stub = PingPongServiceGrpc.newBlockingStub(channel);

        PingRequest request = PingRequest.newBuilder().setMessage("Ping").build();
        PongResponse response;

        response = stub.ping(request);

        logger.info(response.getMessage());
        channel.shutdown();
    }
}

File: ping-pong-grpc/python/pingpong_client.py
import grpc
import pingpong_pb2
import pingpong_pb2_grpc

def run():
  channel = grpc.insecure_channel('localhost:8080')

  stub = pingpong_pb2_grpc.PingPongServiceStub(channel)

  request = pingpong_pb2.PingRequest(message='Ping')

  print("Ping Pong client calling server...")

  response = stub.ping(request)

  print("Response received, message: " + response.message)

if __name__ == '__main__':
  run()